components/brave_rewards/resources/page/components/connect_wallet_modal.tsx

```diff
@@ -155,6 +155,7 @@ export function ConnectWalletModal (props: Props) {
                   setShowMinimumBalanceWarning(true)
                 } else {
                   setShowMinimumBalanceWarning(false)
+                  chrome.send('brave_rewards.setExternalWalletType', [provider.type])
```

Comment:
We shouldn't be calling out to the host from these display components. Instead, we should modify the `onContinue` handler in pageWallet to perform the call.

Reply:
resolved